### PR TITLE
[Bluetooth] migrate to bnd

### PR DESCRIPTION
**Pull request by davidgraeff**
Use maven tinyb for the bluez sub-bundle.

Signed-off-by: David Graeff <david.graeff@web.de>

### CODE REVIEW

**Comment by cweitkamp** [comment]
So, the only test is not worth a separate bundle :wink:? Did you miss the BOM?

**Comment by davidgraeff** [comment]
Tests are ridiculous in this bundle, I agree.
Probably because it came from ESH where every binding was required to have a test bundle.

Yeah I probably neglected the bom. Can I do a follow up PR (would do a github web PR)?

File: bundles/org.openhab.binding.bluetooth.bluez/pom.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <name>openHAB Add-ons :: Bundles :: BlueZ Bluetooth Adapter</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.bluetooth</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+<dependency>
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.bluetooth.bluez/pom.xml]: There is a minor break with formatting too.

File: bundles/org.openhab.binding.bluetooth.bluez/pom.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <groupId>org.sputnikdev</groupId>
+    <artifactId>bluetooth-manager-tinyb</artifactId>
+    <version>1.3.3</version>
+</dependency>
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.bluetooth.bluez/pom.xml]: Same formatting problem here.
Also tinyb dependency should be ` <scope>provided</scope>`

File: features/karaf/openhab-addons/src/main/feature/feature.xml [hunk 3]
```diff
@@ -84,11 +84,12 @@
     <feature name="openhab-binding-bluetooth" description="Bluetooth Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-serial</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.bluetooth/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.bluetooth.blukii/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.bluetooth.ruuvitag/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.bluetooth.bluez/${project.version}</bundle>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.bluetooth.bluegiga/${project.version}</bundle>
+        <bundle dependency="true">wrap:mvn:org.sputnikdev/bluetooth-manager-tinyb/1.3.3$Bundle-Name=Tinyb&amp;Bundle-SymbolicName=org.sputnikdev.bluetooth-manager-tinyb&amp;Bundle-Version=1.3.3</bundle>
```
Comment (Hilbrand) [inline features/karaf/openhab-addons/src/main/feature/feature.xml]: Missing `<feature prerequisite="true">wrap</feature>`

File: bundles/org.openhab.binding.bluetooth.bluez/pom.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.bluetooth</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.sputnikdev</groupId>
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.bluetooth.bluez/pom.xml]: 2 spaces david, 2 :smile:

Comment (davidgraeff) [reply bundles/org.openhab.binding.bluetooth.bluez/pom.xml]: I mean... it looks nice with 4 spaces as well :D Will fix that

**Pull request merged**